default.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs? import <nixpkgs> {}
+, golem-examples ? pkgs.fetchFromGitHub { 
+    owner = "golemcloud";
+    repo = "golem-examples";
+    rev = "5165dcc7e3cbfa09f752caa96a869d284ec169aa";
+    hash = "sha256-7uWvvrRpIo4euFgP0TG4PEXENl4+Wgd8ckPpYnAwQbw=";
+  }
+}:
+pkgs.rustPlatform.buildRustPackage {
+  pname = "golem-cli";
+  version = "0.0.98";
```

Comment:
This is outdated now - who and when would update it? We are not keeping our version numbers anywhere else in our repo but use git tags to do releases.

Reply:
it can be set to 0.0.0, I guess it will be possible to set it in a shellScript command before build from a tag but that will count as kinda of "side effect" and then you will have issue for people that will want to run:
```
nix build github:golemcloud/golem
```
without cloning it.